.github/workflows/lint-clang-formatting.yml

```diff
@@ -0,0 +1,48 @@
+name: Check Code Formatting
```

Comment:
`m`: Are you going to add workflows for `format-markdown`, `format-json`, and `format-yaml` aswell?

Reply:
I think the naming is alright, I just missed that we already have the other two checks.